packages/core/lib/modules/wallet.js

```diff
@@ -0,0 +1,185 @@
+import {WalletRpc} from '../client/wallet-rpc';
+import {v4 as uuid} from 'uuid';
+import {KeyringRpc} from '../client/keyring-rpc';
+import {UtilCryptoRpc} from '../client/util-crypto-rpc';
+import {NetworkManager} from './network-manager';
+import {initRealm} from '../core/realm';
+import {ApiRpc} from '../client/api-rpc';
+import {DockRpc} from '../client/dock-rpc';
+import {EventManager} from './event-manager';
+
+export type DocumentType =
+  | 'mnemonic'
+  | 'credential'
+  | 'account'
+  | 'did'
+  | 'generic';
+
+export const NetworkConfigs = {
```

Comment:
I'd probably pull these out into their own file.

If someone was to build their own full node(s) on the chain for interacting with it how easily could they inject a different configuration? Not saying it's required - just curious.

Reply:
Yeah good point. We can add support for custom network in the network manager.

Reply:
Addd here https://www.notion.so/dockteam/Wallet-sdk-code-review-25d2f2d7644c470ea5a1012a8cd7c871